src/sentry/demo/data_population.py

```diff
@@ -466,20 +508,23 @@ def fix_measurements(event_json):
         measurements.update(measurement_markers)
 
 
-def update_context(event, trace=None):
+def update_context(event, trace=None, mobile=False, platform=None):
```

Comment:
@roggenkemper Maybe instead of passing in a `mobile` argument, we could do a lookup based on the platform since we have to pass that in anyways.

Reply:
good idea!